scripts/run_tests.sh

```diff
@@ -28,7 +28,7 @@ display_result $? 1 "Code style check (Black)"
 flake8 .
 display_result $? 1 "Code style check"
 
-mypy .
+mypy . --exclude build/
```

Comment:
Mypy was scanning the build folder and throwing errors about duplicate modules so it has been excluded.